scoring/src/main/scala/ai/h2o/sparkling/ml/models/PredictionWithContributions.scala

```diff
@@ -33,4 +34,8 @@ trait PredictionWithContributions {
       contributionNames.zip(contributions).toMap
     }
   }
+
+  def supportsContributions(): Boolean = {
+    getWithContributions() && H2OMOJOCache.canGenerateContributions(H2OMOJOCache.getMojoBackend(uid, getMojo, this).m)
```

Comment:
Can we generate warning here? Or better throw an exception that contributions are not supported in the model? I'm afraid that warning will get easily lost in tons of spark logging messages.

Reply:
If we decide to throw an exception then it makes more sense to throw it right away in the `canGenerateContributions` method as that check is done in `loadMojoBackend` before any computation starts.

Reply:
ok, makes sense